modules/installation-user-infra-machines-advanced-customizing-live-iscsi-ibft.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-user-infra-machines-advanced-customizing-live-{boot}-iscsi-ibft_{context}"]
+= Customizing a live install {boot-media} for an iSCSI boot device with iBFT
+
+You can set the iSCSI target and initiator values for automatic mounting, booting and configuration using a customized version of the live {op-system} image.
+
+.Prerequisites
+. You have an iSCSI target you want to install {op-system} on.
+. Optional: you have multipathed your iSCSI target.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+ifeval::["{boot-media}" == "ISO image"]
+. Retrieve the {op-system} ISO image from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/[{op-system} image mirror] page and run the following command to customize the ISO image with the following information:
++
+[source,text]
+----
+$ coreos-installer iso customize \
+    --pre-install mount-iscsi.sh \ <1>
+    --post-install unmount-iscsi.sh \ <2>
+    --dest-device /dev/mapper/mpatha \ <3>
+    --dest-ignition config.ign \ <4>
+    --dest-karg-append rd.iscsi.firmware=1 \ <5>
+    --dest-karg-append rd.multipath=default \ <6>
+    -o custom.iso rhcos-<version>-live.x86_64.iso
+----
+<1> The script that gets run before installation. It should contain the `iscsiadm` commands for mounting the iSCSI target and any commands enabling multipathing.
```

Comment:
Overall looks good to me, one small question, not the blocker, does `mount-iscsi.sh` is like the `iscsiadm` command in https://github.com/openshift/openshift-docs/pull/77275 ?

Reply:
yeah, pretty much! 😄 i think links between the two procedures is a good idea, i'll take a look at adding some post-GA